### PR TITLE
Added logging and shorthand increment

### PR DESCRIPTION
**Pull request by suraneti**
- Removed log `line: 191` because it's log seem like `line: 260` (this may get confused).
- Added query received event log `line:198`.
- Changed `let` to `const` because value never reassigned `line:289` 
- Shorthand increment for cleanly.

### CODE REVIEW

File: libra_service.js [hunk 1]
```diff
@@ -188,14 +188,14 @@ class Libra {
...
     if (event === 'sent') {
-      console.log('sent_events_count', this.sent_events_count)
       if (this.sent_events_count > 0) {
         console.log(`query event ${this.userAddress} ${event} ${this.sent_events_count - 1} false 5\n`)
         await streamWrite(writable, `query event ${this.userAddress} ${event} ${this.sent_events_count - 1} false 5\n`)
         await sleep(1000)
       }
     } else {
       if (this.received_events_count > 0) {
+        console.log(`query event ${this.userAddress} ${event} ${this.received_events_count - 1} false 10\n`)
```
Comment (totiz) [inline libra_service.js]: Could you please make this command as const, and reuse it on streamWrite.

Comment (totiz) [reply libra_service.js]: And perhaps the line above on "if" in the same way as well, thanks.

Comment (suraneti) [reply libra_service.js]: ok, I got it.

**Comment by suraneti** [comment]
@totiz I've updated following your suggestion.

**Comment by totiz** [comment]
Nice!!

**Pull request merged**